test/test_model.py

```diff
@@ -4410,5 +4410,19 @@ def test_error(self, fake_script: FakeScript, model: ops.Model):
         assert str(excinfo.value) == 'ERROR cannot access cloud credentials\n'
 
 
+@pytest.mark.skipif(
+    not hasattr(ops.testing, 'Context'), reason='requires optional ops[testing] install'
+)
+def test_departing_unit_in_relations():
```

Comment:
Does this test succeeding mean that Scenario uses `_make_framework`?

Reply:
Yes. Scenario uses (a subclass of) `_Manager`. As far as possible, it tries to use the same ops mechanics, other than the Juju hook tools.